### PR TITLE
Add NASA

### PR DESCRIPTION
**Pull request by adamalston**
simpleicons

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Alternative to the 'Meatball' logo proposed in #3447

### CODE REVIEW

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:4915]: Thanks @adamalston, though I'm a bit confused on this colour. The SVG itself used `#DB362D`, whereas the [Wikipedia page]() suggests that `#FC3D21` is NASA Red. Taking the RGB values from [their 2006 style guide](https://users.cs.jmu.edu/bernstdh/web/common/policies/NASA_StyleGuide_Nov06.pdf) gives `#FC3D21`, so I think we should go with this.

Comment (adamalston) [reply _data/simple-icons.json:4915]: Per the File History comments on the [Wikipedia page](https://commons.wikimedia.org/wiki/File:NASA_Worm_logo.svg);
> Corrected color to PMC 179, used for "worm" according to this NYT article: https://www.nytimes.com/2020/04/08/science/nasa-logo-worm-spacex.html "In the manual, the color was defined as simply “NASA Red.” Today, there is a universal color matching system defined by the Pantone company, and NASA Red is Pantone color number 179."

Pantone 179 maps to `#E03C31` per the [Pantone website](https://www.pantone.com/color-finder/179-C)

If the suggestion that it is changed still stands, I can easily change it. Just didn't know what the *correct* color was for the svg.

Comment (adamrusted) [reply _data/simple-icons.json:4915]: I'm happy to go with `#E03C31`, being as the 2006 style guide refers to the `meatball` logo instead of the `worm` wordmark. Let me review the path, and will merge if all looks good 👍🏻

File: icons/nasa.svg
Comment (adamrusted) [inline icons/nasa.svg:1]: Other than a load of points in both `A` shapes, this looks spot on. 💯
@adamalston could you have a look at reducing these points down - but no worries if it distorts the icon.

image

Comment (adamalston) [reply icons/nasa.svg:1]: I opened the svg in an editor and I do not see any extraneous points. Can you provide more insight on how to find/reproduce the errors?

Comment (adamrusted) [reply icons/nasa.svg:1]: @adamalston yep, so I've opened the icon in Figma, and double clicked on the vector to show the points. A lot of vector tools have options to simplify paths - but not to worry if you're unable to access it. As I say the icon is spot on, just those points which would be a minor optimization. 💪

Comment (adamalston) [reply icons/nasa.svg:1]: When I simplify the path in the editor, the shape of the letters is distorted. 

If you would like to submit a change, I'm fine with that. Otherwise, I don't know how to fix the extraneous points at the moment.

Comment (adamrusted) [reply icons/nasa.svg:1]: No worries - I'm not able to edit the points in Inkscape, and the icon just won't open in Illustrator! 😅
Happy to merge this as is, unless another @simple-icons/maintainers (like @fbernhart 🧙🏻‍♂️) is able to optimize further. Will wait until later today, just in case they're able to get them.

Comment (ericcornelissen) [reply icons/nasa.svg:1]: >  the icon just won't open in Illustrator!

Chiming in here just to say something about this. This is likely due to the icon being optimized with [SVGOMG](https://jakearchibald.github.io/svgomg/). It does some optimization that Illustrator doesn't like. You can fix this by reoptimizing with our [`.svgo.yml` configuration](https://github.com/simple-icons/simple-icons/blob/3166f14e71bcbdda1a61fc2a6b4513b3ab493982/.svgo.yml). (For more details see https://github.com/simple-icons/simple-icons/issues/2061 and related thread :slightly_smiling_face:)

**Review by adamrusted — APPROVED** [review]
Thanks @ericcornelissen - can verify that I'm unable to simplify by removing those points. Will merge this one for now, and someone else can submit a PR if they're able to optimize further. Thanks again @adamalston for your work on this! 🥳

**Pull request merged**
